blog/drafts-unpublished/announcing-constantinople/README.md

```diff
@@ -0,0 +1,68 @@
+# Post
+
+#### Title
+
+Announcing Constantinople
+
+#### Purpose
+
+Sharing plans for our upcoming Constantinople testnet
+
+#### url
+
+blog.joystream.org/announcing-constantinople/
+
+#### Cover
+
+<p align="center"><img src="announcing-constantinople-cover.png"></p>
+
+#### Lead
+
+Constantinople will be a considerably faster release than Rome, focusing more on governance, and designed to create a more realistic incentive structure.
+
+#### Body
+
+---
+
+## What is Constantinople?
+
+Whereas Rome was a great leap forward in terms of creating a dynamic and flexible content directory for the future, the improvements was rather low level, and not very visible and tangible at this stage. Constantinople on the other hand will introduce features that will immediately impact current users, with two major changes affecting the governance and the economic structure of the platform:
+
+* A proposal system that changes the governance of the platform, by shifting the day to day responsibility of monitoring the network and allocating resources to the Voters and the Council
+* A new incentive structure, with a fiat pool backing the testnet tokens, creating a more realistic economic system for the platform
+
+#### New Proposal System
+Instead of Jsgenesis setting the role parameters, hiring and firing, and running the platforms token economy through a top down approach, Constantinople will introduce a bottom up system where members can make proposal that the Council will vote on
```

Comment:
```suggestion
Rather than Jsgenesis setting the role parameters, hiring and firing, and running the platforms token economy through a top-down approach, Constantinople will introduce a bottom-up system where members can make proposals that the `Council` will vote on
```